docs/services/versioning.mdx

```diff
@@ -50,7 +70,66 @@ curl localhost:9070/deployments --json '{"uri": "http://localhost:9080", "use_ht
 When registering deployments via the UI, you can select the HTTP/1.1 checkbox.
 
 
-## Deploying new service versions
+## Automatic versioning with FaaS platforms
+
+Function-as-a-Service (FaaS) platforms automatically handle immutable versioning through version-specific URLs or ARNs. This makes them ideal for Restate deployments as they eliminate the complexity of manual version management.
+
+<AccordionGroup>
+<Accordion title="AWS Lambda">
+When you [publish a Lambda function](https://docs.aws.amazon.com/lambda/latest/dg/configuration-versions.html), it automatically creates an immutable version with a unique ARN that never changes:
+
+```bash
+# Register a specific Lambda version
+restate deployments register arn:aws:lambda:us-east-1:123456789012:function:my-service:42
+```
+
+Once created, versions remain accessible, so ongoing invocations can continue using them until they finish.
+Old versions automatically phase out once no invocations are using them.
+</Accordion>
+
+<Accordion title="Deno Deploy">
+Deno Deploy creates [immutable deployments](https://docs.deno.com/deploy/manual/deployments/) with unique URLs for each deployment:
+
+```bash
+# Each deployment gets a unique, immutable URL
+restate deployments register https://my-service-abc123.deno.dev
+```
+
+Once created, deployments remain accessible, so ongoing invocations can continue using them until they finish.
+Old deployments automatically phase out once no invocations are using them.
+</Accordion>
```

Comment:
For lambda, deno, vercel, cf workers, can you simply send people to the dedicated pages, rather than duplicating what we already say there?